app/enterprise/1.3-x/developer-portal/structure-and-file-types.md

```diff
@@ -82,58 +100,61 @@ readable_by: [red, blue]
+File contents can be broken down into two parts: `headmatter` and `body`
 
 ##### headmatter
 
-The first thing you will notice in the example files contents are the two sets of `---` delimeters at the start.  The text contained within these markers is called `headmatter` and will always be parsed and validated as valid `yaml`.  `headmatter` contains information necessary for a file to render successfully, as well as any information you would like to access within a template.  Any valid `yaml` key value pair will be parsed by Kong and made available within the contends coinciding html template. There are a few reserved attributes that have special meaning to Kong at time of render. They are as follows:
+The first thing you will notice in the example files contents are the two sets of `---` delimeters at the start.  The text contained within these markers is called `headmatter` and will always be parsed and validated as valid `yaml`.  `headmatter` contains information necessary for a file to render successfully, as well as any information you would like to access within a template.  Any valid `yaml` key value pair will be parsed by Kong and made available within the contents coinciding html template. There are a few reserved attributes that have special meaning to Kong at time of render. They are as follows:
 
-- ###### `title`:
+- `title`:
   - **required**: false
   - **type**: `string`
   - **description**: The title attribute is not necessary but is recommended. When set this will set the title of a page in the browser tab.
   - **example**: `homepage`
-- ###### `layout`
+- `layout`
   - **required**: true
   - **type**: `string`
   - **description**: The layout attribute is required for each piece of content, and determines what html layout to use in order to render the page.  This attribute assumes  a root of the current themes layout directory (`themes/<theme-name>/layouts`).
   - **example**: `bio.html` or `team/about.html`
-- ###### `readable_by`
+- `readable_by`
   - **required**: false
   - **type**: `array` (multiple), `string` (singular)
-  - **description**: The optional readable_by attribute determines which developers can access a piece of content.  In the case of the example above, only developers with rbac roles of "red" or "blue" may access the `/homepage` route.
+  - **description**: The optional `readable_by` attribute determines which developers can access a piece of content.  In the case of the example above, only developers with rbac roles of "red" or "blue" may access the `/homepage` route.
   - **example**: `[red, blue]` (multiple), `red` (singular)
 
-- ###### `route`
+- `route`
   - **required**: false
   - **type**: `string`
   - **description**: This optional attribute overrides the generated route Kong assigns to content, and replaces it with the route included here.
   - **example**: `route: /example/dog` will render the example page above at `<url>/example/dog` instead of the auto generated `/homepage` route.
```

Comment:
```suggestion
  - **example**: `route: /example/dog` renders the example page above at `<url>/example/dog` instead of the auto generated `/homepage` route.
```